carma_nav2_port_drayage_demo/src/port_drayage_demo.cpp

```diff
@@ -63,20 +101,30 @@ auto PortDrayageDemo::on_mobility_operation_received(
   const carma_v2x_msgs::msg::MobilityOperation & msg) -> void
 {
   if (msg.strategy != "carma/port_drayage") {
```

Comment:
Will there ever be a case where mobility operation messages with different strategies will be used together? This could cause some terminal spam if so

Reply:
I left it in, but we should probably remove it. I think it's simple enough to document that and/or check for it in the messages you are publishing. For port drayage, we probably won't have other mobility operation messages but platooning and TIM/TSP use cases do use them if we implement them later.